transport/http_util.go

```diff
@@ -92,7 +92,8 @@ var (
 type decodeState struct {
 	err error // first error encountered decoding
 
-	encoding string
+	acceptEncoding []string
```

Comment:
This doesn't appear to be used by anything.  If there's a longer term plan for this, please add a TODO, or just remove it.